fickling/hook.py

```diff
@@ -1,128 +1,16 @@
 import pickle
 
-import torch
-
 import fickling.loader as loader
-import fickling.polyglot as polyglot
-from fickling.fickle import Pickled
-from fickling.pytorch import PyTorchModelWrapper
 
 
-def hook_pickle_load(pickle_load_function):
+def hook_pickle_load(*args, **kwargs):
+    # TODO Explicitly handle arguments in loader.py such that it can be set in context as well
     # Wraps the function to actually run the core loader
     def wrapper(*args, **kwargs):
-        file = args[0]
-        return loader.load(file)
+        return loader.load(*args, **kwargs)
 
     return wrapper
```

Comment:
Actually since we have `loader.load()` we don't even need that function anymore I think?
We can literally just do 
```
def run_hook():
    pickle.load = fickling.loader.load
```
and that's it :D